tests/spec/dirty_tests.py

```diff
@@ -173,6 +173,29 @@ def it_shows_error_on_stop_for_slow_start(self):
             norm=norm.pgctl,
         )
 
+    def it_restarts_fine_with_force_flag(self):
+        assert_command(
+            ('pgctl', 'start', 'sweet'),
+            '',
+            '''\
+[pgctl] Starting: sweet
+[pgctl] Started: sweet
+''',
+            0,
+        )
+        assert_command(
+            ('pgctl', 'restart', 'sweet', '--force'),
+            '',
+            '''\
+[pgctl] Stopping: sweet
+[pgctl] Stopped: sweet
+[pgctl] Starting: sweet
+[pgctl] Started: sweet
+''',
+            0,
+            norm=norm.pgctl,
+        )
```

Comment:
Please file this under `DescribeSlowShutdown.it_succeeds_with_force()`.
It should be exactly the same as the `it_fails_by_default` test, but showing the difference if we use --force.

I _don't_ believe you should allow pgctl to use kill -9 without throwing some kind of warning.
More specifics below.